internal/tiltfile/tiltfile_test.go

```diff
@@ -4321,6 +4321,52 @@ local_resource('e', 'echo e')
 	}
 }
 
+func TestMaxParallelUpdates(t *testing.T) {
```

Comment:
can you put this test in the subpackage? i don't want this test file to have the union of all extensions